makefiles/buildtests.inc.mk

```diff
@@ -7,17 +7,19 @@ buildtest:
 	@ \
 	RESULT=true ; \
 	for board in $(BOARDS); do \
-		$(COLOR_ECHO) -n "Building for $$board ... " ; \
-		BOARD=$${board} RIOT_CI_BUILD=1 RIOT_VERSION_OVERRIDE=buildtest \
-			$(MAKE) clean all -j $(NPROC) >/dev/null 2>&1; \
-		RES=$$? ; \
-		if [ $$RES -eq 0 ]; then \
-			$(COLOR_ECHO) "$(COLOR_GREEN)success.$(COLOR_RESET)" ; \
-		else \
-			$(COLOR_ECHO) "$(COLOR_RED)failed!$(COLOR_RESET)" ; \
-			RESULT=false ; \
-		fi ; \
-		$(MAKE) clean-intermediates >/dev/null 2>&1 || true; \
+		if $(MAKE) check-toolchain-supported; then
```

Comment:
Needs a terminating `\` and also silencing stdout and stderr. Missing `BOARD=$${board}`.

Suggestion, it could be replace by an `!` condition and using `continue` to remove one indentation.